src/Subscription.re

```diff
@@ -0,0 +1,68 @@
+module type Config = {
+  let query: string;
+  type t;
+  let parse: Js.Json.t => t;
+};
+
+module Make = (Config: Config) => {
+  [@bs.module] external gql: ReasonApolloTypes.gql = "graphql-tag";
+
+  type error = {. "message": string};
+
+  [@bs.deriving abstract]
+  type options = {
+    [@bs.optional]
+    skip: bool,
+    [@bs.optional]
+    onSubscriptionData: unit => unit,
```

Comment:
nice, maybe we can bind already `subscriptionData` and `client` https://github.com/trojanowski/react-apollo-hooks/blob/master/src/useSubscription.ts#L18?

Reply:
Can you explain further what you mean?

Reply:
it's because onSubscriptionData receives an object with two keys `subscriptionData` and `client`

Reply:
I think it's ok for now, I'll merge like this